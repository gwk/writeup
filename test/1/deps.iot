{
  'cmd': '$PROJ/writeup/__main__.py -deps test/1/html/embed.wu',
  'links': 'test',
}
