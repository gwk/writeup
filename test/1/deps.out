test/1/html/test/assets/circle.svg
test/1/html/test/assets/empty.csv
test/1/html/test/assets/head.gif
test/1/html/test/assets/head.jpg
test/1/html/test/assets/head.png
test/1/html/test/assets/line.wu
test/1/html/test/assets/nested.wu
test/1/html/test/assets/table.csv
test/1/html/test/assets/text.txt
